flang/lib/Lower/OpenMP.cpp

```diff
@@ -2963,6 +2963,7 @@ genTargetOp(Fortran::lower::AbstractConverter &converter,
   llvm::SmallVector<mlir::Type> mapSymTypes;
   llvm::SmallVector<mlir::Location> mapSymLocs;
   llvm::SmallVector<const Fortran::semantics::Symbol *> mapSymbols;
+  llvm::SmallVector<const Fortran::semantics::Symbol *> redSymbols;
```

Comment:
```suggestion
  llvm::SmallVector<const Fortran::semantics::Symbol *> reductionSymbols;
```
I think it's best to spell it out, so it's not confusing at first read.

Reply:
Done.